LPS-197429 test

liulukiwi opened the pull request.

victorhcunha committed:
LPS-193134 Update DateTime API request property

alvarosaugarlr committed:
LPS-198665 - add new methods to RemoteService

alvarosaugarlr committed:
LPS-198665 - build service

alvarosaugarlr committed:
LPS-198665 - sf

alvarosaugarlr committed:
LPS-198665 - validation permissions

Commit:
LPS-198665 Think of *ResourceImpl as a presentation layer, and avoid further calls to CommonPermissionUtil outside of AddressServiceImpl

Commit:
LPS-198665 Remove all usage of AddressLocalService

Commit:
LPS-198665 Cannot rely on remote clients to provide correct userId

Commit:
LPS-198665 Update usages

Commit:
LPS-198665 Avoid creating new service methods in 7.4

Commit:
LPS-198665 buildService

victorhcunha committed:
LPS-193134 Fix AssertFieldValuesPersist test

victorhcunha committed:
LPS-193134 Delete unnecessary file

oliverkecskemety committed:
LPS-200030 Add Group schema to SCIM rest-openapi.yaml

regiCesar21 committed:
LPS-200994 move all methods from TimerExecutor and TaskAssignerHelper to the only usage on DefaultWorkflowEngineImpl, and transform some methods into a private one

regiCesar21 committed:
LPS-200994 rename and transform KaleoScriptingEvaluator to BaseKaleoScriptingEvaluator and apply to usages using extends

regiCesar21 committed:
LPS-200994 passing only reference from WorkflowMetricsSLAProcessor by method attribute, apply to usages and test cases

liulukiwi committed:
LPS-197429 test

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.

liulukiwi commented:
ci:test:echo-acceptance

liferay-continuous-integration commented:
<html><h3>:x: ci:test:echo-acceptance - 0 out of 2 jobs passed in 53 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/ee303dbf5d7a3c23dfc86c727bb4c78ed894e749">ee303dbf5d7a3c23dfc86c727bb4c78ed894e749</a></p><h4>Upstream Comparison:</h4><p>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/ee303dbf5d7a3c23dfc86c727bb4c78ed894e749">ee303dbf5d7a3c23dfc86c727bb4c78ed894e749</a><br/>Jenkins Build URL: <a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=2455800447">EE Development Acceptance (master) - 140 - 2023-11-09[04:40:12]</a></p><details><summary><strong>ci:test:echo-acceptance - 0 out of 2 jobs PASSED</strong></summary><div><h4>2 Failed Jobs:</h4><ul><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest(master)/10713/"><strike>test-portal-acceptance-pullrequest(master)</strike></a></li><li>functional-tomcat90-mysql57-jdk8<ul><li><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/933559/"><strike>functional-tomcat90-mysql57-jdk8/0/0</strike></a></li></ul></li></ul></div></details><h5>For more details click <a href="https://test-1-27.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/10713/jenkins-report.html">here</a>.</h5><hr/><h4>Failures unique to this pull:</h4><ol><li><div><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/933559//consoleText">functional-tomcat90-mysql57-jdk8/0/0</a><ol><li><div><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/933559//testReport/com.liferay.poshi.runner/PoshiRunner/test_LocalFile_WebContentUpgradeWithWithFirstComplexStructure_CanDuplicateRepeatableFieldsAndRemoveThem_">LocalFile.WebContentUpgradeWithWithFirstComplexStructure#CanDuplicateRepeatableFieldsAndRemoveThem</a> - <a href="https://testray.liferay.com/reports/production/logs/2023-11/test-1-27/test-portal-acceptance-pullrequest(master)/10713/functional-tomcat90-mysql57-jdk8/0/0/LocalFile.WebContentUpgradeWithWithFirstComplexStructure_CanDuplicateRepeatableFieldsAndRemoveThem/index.html.gz">Poshi Report</a> - <a href="https://testray.liferay.com/reports/production/logs/2023-11/test-1-27/test-portal-acceptance-pullrequest(master)/10713/functional-tomcat90-mysql57-jdk8/0/0/LocalFile.WebContentUpgradeWithWithFirstComplexStructure_CanDuplicateRepeatableFieldsAndRemoveThem/summary.html.gz">Poshi Summary</a> - <a href="https://testray.liferay.com/reports/production/logs/2023-11/test-1-27/test-portal-acceptance-pullrequest(master)/10713/functional-tomcat90-mysql57-jdk8/0/0/LocalFile.WebContentUpgradeWithWithFirstComplexStructure_CanDuplicateRepeatableFieldsAndRemoveThem/console.txt.gz">Poshi Console</a> - <a href="https://testray.liferay.com/reports/production/logs/2023-11/test-1-27/test-portal-acceptance-pullrequest(master)/10713/functional-tomcat90-mysql57-jdk8/0/0/jenkins-console.txt.gz">Console Output</a></div></li></ol></div></li></ol><h4>For upstream results, click <a href="https://test-1-40.liferay.com/job/test-portal-acceptance-upstream-dxp(master)">here</a>.</h4></details></html>

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest(master)/10713/">test-portal-acceptance-pullrequest(master)#10713</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-27.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/10713/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>echo-acceptance<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/liulukiwi/liferay-portal/pull/452">liulukiwi#452</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/builds?testrayRoutineId=1306753894">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=2456449331">[master] ci:test:echo-acceptance - liulukiwi &gt; liulukiwi - PR#452 - 2023-11-10[02:33:49]</a><br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-27.liferay.com/job/publish-testray-report/23110/">publish-testray-report#23110</a><br/></div></div>

DiegoHu97 committed:
LPS-201214 Replace AUI() usages when exporting a Kaleo Process

DiegoHu97 committed:
LPS-201214 Replace AUI() usages when changing Kaleo Form fields

DiegoHu97 committed:
LPS-201214 Replace AUI() usages when changing Kaleo assigned form

DiegoHu97 committed:
LPS-201214 Remove not used code

liulukiwi committed:
LPS-197429 just test

liulukiwi commented:
ci:test:echo-acceptance

liferay-continuous-integration commented:
<html><h3>:x: ci:test:echo-acceptance - 0 out of 2 jobs passed in 46 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/c9ef29dd3573a3961c0d4086138b92d710c4422d">c9ef29dd3573a3961c0d4086138b92d710c4422d</a></p><h4>Upstream Comparison:</h4><p>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/c9ef29dd3573a3961c0d4086138b92d710c4422d">c9ef29dd3573a3961c0d4086138b92d710c4422d</a><br/>Jenkins Build URL: <a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=2457194375">EE Development Acceptance (master) - 144 - 2023-11-11[08:39:34]</a></p><details><summary><strong>ci:test:echo-acceptance - 0 out of 2 jobs PASSED</strong></summary><div><h4>2 Failed Jobs:</h4><ul><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest(master)/11921/"><strike>test-portal-acceptance-pullrequest(master)</strike></a></li><li>functional-tomcat90-mysql57-jdk8<ul><li><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/692010/"><strike>functional-tomcat90-mysql57-jdk8/0/0</strike></a></li></ul></li></ul></div></details><h5>For more details click <a href="https://test-1-26.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/11921/jenkins-report.html">here</a>.</h5><hr/><h4>Failures unique to this pull:</h4><ol><li><div><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/692010//consoleText">functional-tomcat90-mysql57-jdk8/0/0</a><ol><li><div><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/692010//testReport/com.liferay.poshi.runner/PoshiRunner/test_LocalFile_WebContentUpgradeWithWithFirstComplexStructure_CanDuplicateRepeatableFieldsAndRemoveThem_">LocalFile.WebContentUpgradeWithWithFirstComplexStructure#CanDuplicateRepeatableFieldsAndRemoveThem</a> - <a href="https://testray.liferay.com/reports/production/logs/2023-11/test-1-26/test-portal-acceptance-pullrequest(master)/11921/functional-tomcat90-mysql57-jdk8/0/0/LocalFile.WebContentUpgradeWithWithFirstComplexStructure_CanDuplicateRepeatableFieldsAndRemoveThem/index.html.gz">Poshi Report</a> - <a href="https://testray.liferay.com/reports/production/logs/2023-11/test-1-26/test-portal-acceptance-pullrequest(master)/11921/functional-tomcat90-mysql57-jdk8/0/0/LocalFile.WebContentUpgradeWithWithFirstComplexStructure_CanDuplicateRepeatableFieldsAndRemoveThem/summary.html.gz">Poshi Summary</a> - <a href="https://testray.liferay.com/reports/production/logs/2023-11/test-1-26/test-portal-acceptance-pullrequest(master)/11921/functional-tomcat90-mysql57-jdk8/0/0/LocalFile.WebContentUpgradeWithWithFirstComplexStructure_CanDuplicateRepeatableFieldsAndRemoveThem/console.txt.gz">Poshi Console</a> - <a href="https://testray.liferay.com/reports/production/logs/2023-11/test-1-26/test-portal-acceptance-pullrequest(master)/11921/functional-tomcat90-mysql57-jdk8/0/0/jenkins-console.txt.gz">Console Output</a></div></li></ol></div></li></ol><h4>For upstream results, click <a href="https://test-1-40.liferay.com/job/test-portal-acceptance-upstream-dxp(master)">here</a>.</h4></details></html>

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest(master)/11921/">test-portal-acceptance-pullrequest(master)#11921</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-26.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/11921/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>echo-acceptance<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/liulukiwi/liferay-portal/pull/452">liulukiwi#452</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/builds?testrayRoutineId=1306753894">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=2457624220">[master] ci:test:echo-acceptance - liulukiwi &gt; liulukiwi - PR#452 - 2023-11-13[00:56:17]</a><br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-26.liferay.com/job/publish-testray-report/27474/">publish-testray-report#27474</a><br/></div></div>

DiegoHu97 committed:
LPS-201214 Replace AUI() usages when changing Kaleo workflow

veroglez committed:
LPS-202697 Add getFieldMap to get a map with the description, title and friendlyURL fields

veroglez committed:
LPS-202697 Add getLanguages to get the necessary languages for the translation manager

victorhcunha committed:
LPS-202741 Fix GO_TO_FOLDER path

veroglez committed:
LPS-202697 Add new parameter for input localized to hide the translation manager

veroglez committed:
LPS-202697 Add new Translation Selector component

victorhcunha committed:
LPS-202741 Fix CanMoveObjectDefinitionToCurrentFolder test

victorhcunha committed:
LPS-202741 Add OBJECT_IN_LEFT_SIDEBAR_KEBAB_MENU path

victorhcunha committed:
LPS-202741 Remove unused macro and path

victorhcunha committed:
LPS-202741 Update SELECT_OBJECT_IN_LEFT_SIDEBAR path to use folder label

victorhcunha committed:
LPS-202741 Apply

veroglez committed:
LPS-202697 Add label for the translation status

veroglez committed:
LPS-202697 Add Translation Manager component

veroglez committed:
LPS-202697 Include Translation Manager component next to the input localized

MarinhoFeliphe committed:
LPS-197066 Pass dtoConverterContext as an argument

MarinhoFeliphe committed:
LPS-197066 Handle picklist fields on DTO parsing

MarinhoFeliphe committed:
LPS-197066 Move the logic to convert the payload to object entry dto into ObjectEntryDTOConverter

veroglez committed:
LPS-202697 Add improvements for accessibility

veroglez committed:
LPS-202697 BuildLang

BarbaraCabrera committed:
LPS-202233 Create KBConfigurationDisplayContext

BarbaraCabrera committed:
LPS-202233 Add necessary methods to the configuration Display Context

BarbaraCabrera committed:
LPS-202233 Add new configuration jsp to show the new view and add this view under feature flag in AdminConfigurationAction

BarbaraCabrera committed:
LPS-202233 Add the necessary elements for the new view and the necessary imports

BarbaraCabrera committed:
LPS-202233 Move saveConfiguration code to propsTransformer

BarbaraCabrera committed:
LPS-202233 Validate just the necessary elementParam

alessiorendina committed:
COMMERCE-12502 Set feature flag to release type

alessiorendina committed:
COMMERCE-12502 Added title and description keys

alessiorendina committed:
COMMERCE-12502 BuildLang

evanilson-santana committed:
LPS-193134 Add WaitForVisible macro to make sure object definition imported

victorhcunha committed:
LPS-203256 Fix CanUpdateOrderAfterUpdatingCommerceOrderEntry test

JanisZhang committed:
LPS-203461 Remove messaging-misc-spring.xml from portal-impl

# breaking

## What portal-impl/src/META-INF/messaging-misc-spring.xml

messaging-misc-spring.xml has been removed.

## Why

No one is using this Spring config anymore.

----

JanisZhang committed:
LPS-203461 Remove messaging-misc-spring.xml from spring.configs

markomihalovics committed:
LPS-203529 Created the RequiredAssetListEntryException class.

markomihalovics committed:
LPS-203529 Extended the Language.properties with the 'this-asset-cannot-be-deleted-because-it-is-in-use' error message.

markomihalovics committed:
LPS-203529 Created the AssetListEntryServiceImpl._checkCompanyAssetListEntryUsages method. Extended the logic of the AssetListEntryServiceImpl.deleteAssetListEntries and AssetListEntryServiceImpl.deleteAssetListEntry methods with the usage of the new method.

markomihalovics committed:
LPS-203529 Imported the RequiredAssetListEntryException in the asset-list-web init.jsp. Added a custom error tag to the asset-list-web view.jsp.

markomihalovics committed:
LPS-203529 Extended the logic of the AssetListEntryServiceImpl.deleteAssetListEntry method with the usage of the AssetListEntryServiceImpl._checkCompanyAssetListEntryUsages method.

victorhcunha committed:
LPS-193134 Fix exportObject macro

victorhcunha committed:
LPS-193134 Improve importObject macro to import files from temp directory

victorhcunha committed:
LPS-193134 Apply

jeffwu0724 committed:
LPS-201589 add a property LiferayActionResponse.SKIP_ESCAPE_REDIRECT to make sure only escape the unauthorized external site

kyle-miho committed:
LRQA-83508 Remove an infinite circular loop when calling BuildReportFactory.newTopLevelBuildReport with a URL parameter

kyle-miho committed:
LRQA-83508 Fully update GenerateTestrayCSV Job to more accurately detect unique failures

brianikim committed:
COMMERCE-12979 Handle view when facets are empty

shuyangzhou committed:
LPS-203913 Decouple ElasticsearchConnectionManager from ClusterExecutor

brianikim committed:
COMMERCE-12979 Do not add to facet list if term results are empty.

peerkar committed:
LPS-203720 portal-search-api: Remove the Category Facet Field configuration

# breaking

## What modules/apps/portal-search/portal-search-api/src/main/java/com/liferay/portal/search/configuration/CategoryFacetFieldConfiguration.java

assetVocabularyCategoryIds will now be used as the aggregation field for the Category Facet.

## Why

In order to implement the Category Facet enhancements described in LPS-156503, a new field assetVocabularyCategoryIds needed to be introduced. If the Category Facet is not showing the categories after the upgrade, a full re-index is required.

----

peerkar committed:
LPS-203720: Remove usages

peerkar committed:
LPS-203720 portal-language-lang: Remove the language keys

Commit:
LPS-201214 Modify macros to adapt new markup

jkappler committed:
LPS-202233 Use same logic as with other configuration actions

jkappler committed:
LPS-202233 Adds missing validations

jkappler committed:
LPS-202233 Move to backend

jkappler committed:
LPS-202233 Adds missing errors

Mikellorza committed:
LPS-203717 Add error message for LayoutTypeException.FIRST_LAYOUT_PERMISSION exception

jkappler committed:
LPS-202233 Not needed

joseabelenda committed:
LPS-203751 Create liferay-testray-site-initializer in workspace

joseabelenda committed:
LPS-203751 Create liferay-testray-custom-element in workspace

Mikellorza committed:
LPS-203757 Fix first layout permission error message since in the type request parameter there is not the type of the layout.

Mikellorza committed:
LPS-203717 Add language key

CrescenzoRegaLF committed:
COMMERCE-12921 Update permissions

CrescenzoRegaLF committed:
COMMERCE-12921 Upgrade process

shuyangzhou committed:
LPS-203913 Fix unit test

joseabelenda committed:
LPS-203751 Move to workspace

pyoo47 committed:
LRQA-83508 Don't throw or catch the generic class, Exception

pyoo47 committed:
LRQA-83508 Rename

pyoo47 committed:
LRQA-83508 Simplify

pyoo47 committed:
LRQA-83508 Sort

pyoo47 committed:
LRQA-83508 typo

shuyangzhou committed:
LPS-203913 Consume the connection to avoid racing assertion.

john-co committed:
LPS-2009346 Use other macros to prevent stalling on log in

john-co committed:
LPS-200946 Have proper teardown by adding portal settings teardown before instance teardown

JanisZhang committed:
LPS-203461 Move LiveUsersMessageListener to module users-admin-web

JanisZhang committed:
LPS-203461 Make LiveUsersMessageListener as a component and register Destination in activate method.

JanisZhang committed:
LPS-203461 Move ClusterBridgeMessageListener to module user-admin-web since it only used for the live_users destination.

# breaking

## What portal-kernel/src/com/liferay/portal/kernel/cluster/messaging/ClusterBridgeMessageListener.java

ClusterBridgeMessageListener was moved to module user-admin-web

## Why

ClusterBridgeMessageListener only used for the live_users destination.

----

JanisZhang committed:
LPS-203461 Remove Bean ClusterBridgeMessageListener and LiveUsersMessageListener from messaging-misc-spring.xml

peerkar committed:
LPS-203720 portal-search-api: gw baseLine

peerkar committed:
LPS-203720 portal-search-test: Update tests

peerkar committed:
LPS-203720 portal-search-web: Update tests

Commit:
LPS-198665 SF

pyoo47 committed:
LRQA-83508 Reorganize, Simplify

ealonso committed:
LPS-203529 Baseline

ealonso committed:
LPS-203529 buildLang

ealonso committed:
LPS-203529 Rephrase

pyoo47 committed:
LRQA-83508 Prevent accidental inclusion of _CSV_DELMITER in data

ealonso committed:
LPS-203529 Simplify

ealonso committed:
LPS-203529 Rename

pyoo47 committed:
LRQA-83508 Cache TestrayCaseResult type

ealonso committed:
LPS-203529 Check if we can delete de AssetListEntry in local method, also deleting a AssetListEntrySegmentsEntryRel doesn't means that we are deleting a collection

ealonso committed:
LPS-203529 Check only if we are not deleting a group

marcosapmf committed:
LRAC-14091 delete those segments from portal that were deleted from AC as well

marcosapmf committed:
LRAC-14091 create test

abfranca committed:
ISSD-1604 New expando columns for menu items

pyoo47 committed:
LRQA-83508 Add DID_NOT_RUN results data to report

shuyangzhou committed:
LPS-203939 Cleanup unnecessary ModelIndexerWriterContributor service wiring in account-service

pyoo47 committed:
LRQA-83508 Invert distance logic

shuyangzhou committed:
LPS-203939 Cleanup unnecessary ModelIndexerWriterContributor service wiring in address-impl

shuyangzhou committed:
LPS-203939 Cleanup unnecessary ModelIndexerWriterContributor service wiring in asset-categories-service

shuyangzhou committed:
LPS-203939 Cleanup unnecessary ModelIndexerWriterContributor service wiring in asset-tags-service

shuyangzhou committed:
LPS-203939 Cleanup unnecessary ModelIndexerWriterContributor service wiring in blogs-service

shuyangzhou committed:
LPS-203939 Cleanup unnecessary ModelIndexerWriterContributor service wiring in bookmarks-service

shuyangzhou committed:
LPS-203939 Cleanup unnecessary ModelIndexerWriterContributor service wiring in calendar-service

shuyangzhou committed:
LPS-203939 Cleanup unnecessary ModelIndexerWriterContributor service wiring in change-tracking-service

shuyangzhou committed:
LPS-203939 Cleanup unnecessary ModelIndexerWriterContributor service wiring in commerce-currency-service

shuyangzhou committed:
LPS-203939 Cleanup unnecessary ModelIndexerWriterContributor service wiring in commerce-inventory-service

shuyangzhou committed:
LPS-203939 Cleanup unnecessary ModelIndexerWriterContributor service wiring in commerce-order-rule-service

shuyangzhou committed:
LPS-203939 Cleanup unnecessary ModelIndexerWriterContributor service wiring in commerce-payment-service

shuyangzhou committed:
LPS-203939 Cleanup unnecessary ModelIndexerWriterContributor service wiring in commerce-product-service

shuyangzhou committed:
LPS-203939 Cleanup unnecessary ModelIndexerWriterContributor service wiring in commerce-product-type-grouped-service

ealonso committed:
LRAC-14091 Simplify

ealonso committed:
LRAC-14091 Reword

Commit:
LPS-198665 baseline

ealonso committed:
LRAC-14091 Uses SegmentsTestUtil

Hijky committed:
LPS-203496 Add a test to get the fields annotated with GraphQLTypeExtension in GraphQL

peerkar committed:
LPS-203720 portal-search-api: gw buildLang

Mikellorza committed:
LPS-203717 buildLang

oliverkecskemety committed:
LPS-200030 gw buildREST

abfranca committed:
ISSD-1604 Rearrange navigation menu via widget template

abfranca committed:
ISSD-1604 Change expando values from navigation menu items

abfranca committed:
ISSD-1604 Auto SF

michaelhashimoto committed:
LRCI-3935 Add Maintenance & Verification to the Liferay databases

john-co committed:
LPS-203426 Isolate test in single run

michaelhashimoto committed:
LRCI-3935 Add Maintenance & Verification job entity implementations

michaelhashimoto committed:
LRCI-3935 Add Maintenance & Verification job definition implementations

michaelhashimoto committed:
LRCI-3935 Add Maintenance & Verification job definition fields

michaelhashimoto committed:
LRCI-3935 Add Maintenance & Verification job entity field & build parameters

jeffwu0724 committed:
LPS-201589 Semantic Versioning

joseabelenda committed:
ISSD-1604 SF

joseabelenda committed:
ISSD-1604 Rename

shuyangzhou committed:
LPS-203939 Cleanup unnecessary ModelIndexerWriterContributor service wiring in commerce-service

shuyangzhou committed:
LPS-203939 Cleanup unnecessary ModelIndexerWriterContributor service wiring in commerce-shop-by-diagram-service

shuyangzhou committed:
LPS-203939 Cleanup unnecessary ModelIndexerWriterContributor service wiring in commerce-term-service

shuyangzhou committed:
LPS-203939 Cleanup unnecessary ModelIndexerWriterContributor service wiring in contacts-service

shuyangzhou committed:
LPS-203939 Cleanup unnecessary ModelIndexerWriterContributor service wiring in data-engine-service

shuyangzhou committed:
LPS-203939 Cleanup unnecessary ModelIndexerWriterContributor service wiring in depot-service

shuyangzhou committed:
LPS-203939 Cleanup unnecessary ModelIndexerWriterContributor service wiring in document-library-service

shuyangzhou committed:
LPS-203939 Cleanup unnecessary ModelIndexerWriterContributor service wiring in dynamic-data-lists-service

shuyangzhou committed:
LPS-203939 Cleanup unnecessary ModelIndexerWriterContributor service wiring in dynamic-data-mapping-service

shuyangzhou committed:
LPS-203939 Cleanup unnecessary ModelIndexerWriterContributor service wiring in export-import-service

shuyangzhou committed:
LPS-203939 Cleanup unnecessary ModelIndexerWriterContributor service wiring in journal-service

shuyangzhou committed:
LPS-203939 Cleanup unnecessary ModelIndexerWriterContributor service wiring in layout-impl

shuyangzhou committed:
LPS-203939 Cleanup unnecessary ModelIndexerWriterContributor service wiring in list-type-service

shuyangzhou committed:
LPS-203939 Cleanup unnecessary ModelIndexerWriterContributor service wiring in message-boards-service

shuyangzhou committed:
LPS-203939 Cleanup unnecessary ModelIndexerWriterContributor service wiring in notification-service

shuyangzhou committed:
LPS-203939 Cleanup unnecessary ModelIndexerWriterContributor service wiring in notifications-service

shuyangzhou committed:
LPS-203939 Cleanup unnecessary ModelIndexerWriterContributor service wiring in object-service

shuyangzhou committed:
LPS-203939 Cleanup unnecessary ModelIndexerWriterContributor service wiring in organizations-service

shuyangzhou committed:
LPS-203939 Cleanup unnecessary ModelIndexerWriterContributor service wiring in portal-workflow-kaleo-service

shuyangzhou committed:
LPS-203939 Cleanup unnecessary ModelIndexerWriterContributor service wiring in redirect-service

shuyangzhou committed:
LPS-203939 Cleanup unnecessary ModelIndexerWriterContributor service wiring in segments-service

shuyangzhou committed:
LPS-203939 Cleanup unnecessary ModelIndexerWriterContributor service wiring in translation-service

shuyangzhou committed:
LPS-203939 Cleanup unnecessary ModelIndexerWriterContributor service wiring in user-groups-admin-impl

shuyangzhou committed:
LPS-203939 Cleanup unnecessary ModelIndexerWriterContributor service wiring in users-admin-impl

shuyangzhou committed:
LPS-203939 Cleanup unnecessary ModelIndexerWriterContributor service wiring in search-experiences-service

Luizcarlosqueiroz committed:
LPS-203908 Fix behavior from account tests and active them

Luizcarlosqueiroz committed:
LPS-203908 Add steps to allow entry deletion and active test

Luizcarlosqueiroz committed:
LPS-203908 Active passing poshi tests

michaelhashimoto committed:
LRCI-3956 Move logic for invoking all jobs into a separate method

michaelhashimoto committed:
LRCI-3956 Add broadcast message upon startup if ci.properties is configured

michaelhashimoto committed:
LRCI-3957 Add auto comment message upon startup if ci.properties is configured

susanchen3 committed:
LPS-199110 Add PostgreSQL conditional

susanchen3 committed:
LPS-199110 Rename to generalize sql statements

susanchen3 committed:
LPS-199110 Run DB Partitioning tests on MySQL and PostgreSQL

susanchen3 committed:
LPS-199110 Temp do not run DB Paritioning tests on PostgreSQL until LPS-198753

susanchen3 committed:
LPS-199110 Only update driver when using MySQL and liferay online properties

susanchen3 committed:
LPS-199110 Ensure MySQL test batches only run MySQL tests

kyle-miho committed:
LRQA-83508 Add fallback incase build report is missing

pyoo47 committed:
LRQA-83508 Simplify

pyoo47 committed:
LRQA-83508 Return "Unknown" instead of the empty string

pyoo47 committed:
LRQA-83508 Catch IOException instead of Exception

qz1010 committed:
ISSD-400 SF

michaelhashimoto committed:
LRCI-3963 Add GitHubOrganization object

michaelhashimoto committed:
LRCI-3963 Add logic to determine if GitHubUser is part of Liferay

michaelhashimoto committed:
LRCI-3963 Prevent tests from running if GitHubUser is not a part of Liferay

vicnate5 committed:
LPS-199110 SF

Commit:
LPS-203691 Remove tear down block

Commit:
LPS-203691 Modify case to adapt new behavior

jaydawu committed:
COMMERCE-12979 Fix a test according to the pr

Hijky committed:
LPS-203881 Test fix for creating an API endpoint

dantewang committed:
LPS-200994 SF, move package

Commit:
POSHI-687 Reassign value

JamesAGarcia committed:
COMMERCE-12502 wordsmiths language keys

JamesAGarcia committed:
COMMERCE-12502 adds title and description to portal.properties

kenjiheigel committed:
LRCI-3973 Allow 'playwright.test.project' to be set per testsuite & batch

brianchandotcom committed:
LRQA-83508 Sort

brianchandotcom committed:
LRQA-83508 prep next

carolmariaabb committed:
LPS-203904 Avoid NullPointerException

ealonso committed:
LPS-203529 Add it to service.xml

ealonso committed:
LPS-203529 Auto

ealonso committed:
LPS-203529 Grammar

ealonso committed:
LPS-203529 buildLang

davisaints committed:
LPS-200779 Do not run TaskReassignedToMultipleUsers in a virtual instance

ealonso committed:
LRAC-14091 It should be segments instead of segment, also adds segments entry entry instead of segment

abfranca committed:
ISSD-1604 Adjust product cards template

ling-alan-huang committed:
ISSD-400 SF

victorhcunha committed:
LPS-203256 Fix CanCreateActionUsingGroovyScripWithOnOrderStatusUpdate test

victorhcunha committed:
LPS-203256 Remove workaround because the test no longer runs on a virtual instance

victorhcunha committed:
LPS-203964 Fix CanUseUniqueValue test

victorhcunha committed:
LPS-199239 Fix FIELD_ATTACHMENTS_OBJECT_SELECT_FIELD path

victorhcunha committed:
LPS-199239 Apply

victorhcunha committed:
LPS-199239 Fix CanDeleteAttachment test

shuyangzhou committed:
LPS-203980 Create org.apache.aries.jax.rs.whiteboard patch module.

shuyangzhou committed:
LPS-203980 Initial checkin

shuyangzhou committed:
LPS-203980 Apply usage

shuyangzhou committed:
LPS-203980 Patch to defer OSGi DSL opening until receives trigger.

shuyangzhou committed:
LPS-203980 Start/stop whiteboard from JAXRSLifecycle

shuyangzhou committed:
LPS-203980 Use Snapshot to get access to JaxrsServiceRuntime

shuyangzhou committed:
LPS-203980 Mute expected error log.

brianchandotcom committed:
LPS-203980 prep next

brianchandotcom committed:
LRCI-3963 Wordsmith/SF

brianchandotcom committed:
LRCI-3956 Wordsmith

brianchandotcom committed:
LPS-197066 Rename

brianchandotcom committed:
Ran ../gradlew -b releng.gradle cleanReleng

brianchandotcom committed:
LPS-203529 Use the new key

brianchandotcom committed:
LPS-203529 SF

brianchandotcom committed:
LPS-203717 Wordsmith

brianchandotcom committed:
LPS-203717 Regen

brianchandotcom committed:
LPS-199110 SF

brianchandotcom committed:
COMMERCE-12921 SF

brianchandotcom committed:
LPS-202233 SF

brianchandotcom committed:
LPS-200030 you're getting more than 1

brianchandotcom committed:
LPS-200030 Regen

brianchandotcom committed:
LPS-202697 Not needed

brianchandotcom committed:
COMMERCE-12502 See https://capitalizemytitle.com/style/APA/

brianchandotcom committed:
LPS-201589 Prefix with LIFERAY

brianchandotcom committed:
U108

liulukiwi committed:
LPS-194486 test
